### PR TITLE
AmpTask1786_Integrate_20211111

### PR DESCRIPTION
**Pull request by gpsaggese**

### CODE REVIEW

File: core/test/TestComputeMaxDrawdown.test1/output/test.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+                  arma(1,1)
```
Comment (gitpaulsmith) [inline core/test/TestComputeMaxDrawdown.test1/output/test.txt:1]: Please remove `core/test/TestComputeMaxDrawdown.test1/output/test.txt `, `core/test/TestComputeMaxDrawdown.test2/output/test.txt` before merging.

**Review by gitpaulsmith — CHANGES_REQUESTED** [review]

File: helpers/hpandas.py [hunk 2]
```diff
@@ -75,3 +78,90 @@ def dassert_valid_remap(to_remap: List[str], remap_dict: Dict[str, str]) -> None
...
+        )
+    else:
+        _LOG.info(
+            "Index length=%s has not changed",
+            len(index)
+        )
+    return resampled_index
+
+
+def resample_df(df: pd.DataFrame, frequency: str) -> pd.DataFrame:
```
Comment (gitpaulsmith) [inline helpers/hpandas.py:130]: We already have a number of specialized resampling functions. Do we need these too?

File: core/test/TestComputeMaxDrawdown.test1/output/test.txt (hunk 1 above)
Comment (gpsaggese) [reply core/test/TestComputeMaxDrawdown.test1/output/test.txt:1]: Will do. Thanks for taking a look. All tests are passing but I'll take another look before merging.

File: helpers/hpandas.py (hunk 2 above)
Comment (gpsaggese) [reply helpers/hpandas.py:130]: It wasn't unclear if the overlap was perfect.

I wanted you to review this code (for this reason I'm merging it back).

I'll file a bug so we can review and consolidate.

Comment (gpsaggese) [reply helpers/hpandas.py:130]: Filed https://github.com/alphamatic/amp/issues/1842

File: core/test/TestComputeMaxDrawdown.test1/output/test.txt (hunk 1 above)
Comment (gpsaggese) [reply core/test/TestComputeMaxDrawdown.test1/output/test.txt]: Removed

**Comment by gitpaulsmith** [comment]
AFAIK we don’t have a check that fails when we have extra “unused” test
files. I paid off some debt earlier this week and in touching those tests
moved them from files to an in-line assertion.

**Pull request merged**